### PR TITLE
fix strong and emphasis

### LINKED ISSUE #5: markdownfmt removing all emphasis
Revision: 429996b

Example input, test.md:

```
_transaction_

*transaction*

**transaction**
```

Command: cat test.md | ./markdownfmt

Expected behavior: the output is identical as input.

Actual behavior: output

```
transaction

transaction

transaction
```

### PR DESCRIPTION
**Pull request by karelbilek**
fixes https://github.com/Kunde21/markdownfmt/issues/5

### CODE REVIEW

File: markdown/main.go [hunk 1]
```diff
@@ -302,27 +302,20 @@ func (_ *markdownRenderer) codeSpan(out *bytes.Buffer, text []byte) {
 	out.WriteByte('`')
 }
 
-func (mr *markdownRenderer) doubleEmphasis(out *bytes.Buffer, text []byte) {
-	//text := node.Literal
-	if mr.opt.Terminal {
-		out.WriteString("\x1b[1m") // Bold.
+func (mr *markdownRenderer) doubleEmphasis(out *bytes.Buffer, node *blackfriday.Node, content []byte) {
+	text := string(content)
+	if len(text) == 0 {
```
Comment (Kunde21) [inline markdown/main.go]: We can collapse these into `len(content) == 0`

File: markdown/main.go [hunk 2]
```diff
@@ -302,27 +302,20 @@ func (_ *markdownRenderer) codeSpan(out *bytes.Buffer, text []byte) {
...
-	if mr.opt.Terminal {
-		out.WriteString("\x1b[0m") // Reset.
-	}
 }
 
-func (_ *markdownRenderer) emphasis(out *bytes.Buffer, text []byte) {
-	//text := node.Literal
+func (_ *markdownRenderer) emphasis(out *bytes.Buffer, node *blackfriday.Node, content []byte) {
+	text := string(content)
 	if len(text) == 0 {
```
Comment (Kunde21) [inline markdown/main.go]: ditto `len(content) == 0`

**Review by Kunde21 — APPROVED** [review]
Minor cleanliness fix.  
Thank you!

**Comment by karelbilek** [comment]
I noticed a bug I introduced :) and fixed it

The tests are failing btw, but they are failing on master too

**Comment by karel-3d** [comment]
I noticed a bug I introduced :) and fixed it

The tests are failing btw, but they are failing on master too

**Review by Kunde21 — APPROVED** [review]

**Pull request merged**